hyper/pkg/proto/hyper.proto

```diff
@@ -5,8 +5,13 @@ option go_package = "github.com/isard-vdi/isard/hyper/pkg/proto";
 
 service Hyper {
     rpc DesktopStart(DesktopStartRequest) returns (DesktopStartResponse) {}
+    rpc DesktopState(DesktopStateRequest) returns (DesktopStateResponse) {}
     rpc DesktopValidate(DesktopValidateRequest) returns (DesktopValidateResponse) {}
     rpc DesktopStop(DesktopStopRequest) returns (DesktopStopResponse) {}
+    rpc DesktopResume(DesktopResumeRequest) returns (DesktopResumeResponse) {}
+    rpc DesktopSuspend(DesktopSuspendRequest) returns (DesktopSuspendResponse) {}
```

Comment:
`DesktopSuspend` should be above `DesktopResume`

Reply:
ok